packages/server-wallet/src/protocols/channel-opener.ts

```diff
@@ -38,34 +38,61 @@ export class ChannelOpener {
       if (!channel) {
         throw new Error(`ChannelOpener can't find channel with id ${channelToLock}`);
       }
+      // always queue the channel if we've potentially touched it
+      response.queueChannel(channel);
```

Comment:
What's the purpose of this addition? Is it necessary for this PR, or was it just noticed as an omission?

Oh, nevermind, I see it was moved up from a conditional block down below.